src/Resources/public/sonata-page.back.js

```diff
@@ -571,32 +586,36 @@
                         'value': !enabled
                     },
                     success: function (resp) {
-                        $childBlock.attr('data-block-enabled', enabled ? "0" : "1");
-                        enabled = !enabled;
-                        $switchButton.toggleClass('bg-yellow bg-green');
-                        $switchBtnIcon.toggleClass('fa-toggle-off fa-toggle-on');
-
-                        if (enabled) {
-                            $switchButton.html($switchLblDsbl);
-                        } else {
-                            $switchButton.html($switchLblEnbl);
-                        }
+                        if (resp.status && resp.status === 'OK') {
```

Comment:
That does not look like "just a formatting change"

Reply:
You're right. Looks like it was introduced here (from blame); https://github.com/sonata-project/SonataPageBundle/commit/c2112c0ed4f7278cb38d629c153ca0296f33529f